packages/sanity/src/core/store/_legacy/datastores.ts

```diff
@@ -155,6 +156,19 @@ export function useDocumentStore(): DocumentStore {
     [telemetry],
   )
 
+  const handleReportLatency = useCallback(
+    (event: LatencyReportEvent) => {
+      if (event.latencyMs > 1000) {
```

Comment:
Move into constant(Could belong with the telemetry event? 🤔) and maybe describe why and what this number is? As a reference and help for our future selves 🙃

Reply:
Good call! Extracted to a const in [5f918c7](https://github.com/sanity-io/sanity/commit/5f918c7a6e7ab8b915ee544d2868ea4801218602)